floss/language/go/extract.py

```diff
@@ -45,6 +45,9 @@ def extract_go_strings(
             b"\x48\xba(........)|\x48\xb8(........)|\x81\x78\x08(....)|\x81\x79\x08(....)|\x66\x81\x78\x0c(..)|\x66\x81\x79\x0c(..)|\x80\x78\x0e(.)|\x80\x79\x0e(.)"
         )
 
+        longstring = re.compile(b"\x48\x8D\x1D(....)\xB9(....)")
+        longstring2 = re.compile(b"\x48\x83\xFB(.)(.){2,5}\x48\x8D\x1D(....)")
```

Comment:
please add example byte sequences or explanations what these byte patterns reflect so we can still understand them in the future (here and above)